### DISCUSSION TITLE
maddy 0.4.4

### DISCUSSION THREAD

**Discussion by foxcpp** [emersion/maddy#349] [General]
> *The* long awaited release!

### Outbound SMTP

* target/remote: Force MX domain to be FQDN when looking up TLSA records (GH #321)
* Fix two issues in handling of DSN messages in SMTP pipeline and checks (GH #327)
* dns: Attempt to use 127.0.0.1 if no DNS servers are configured in system
* target/queue: Do not attempt to do atomic overwrite for metadata on Windows (GH #334)

### Inbound SMTP

* endpoint/smtp: Unbreak `MAIL FROM:<>` handling (GH #337)
* endpoint/smtp: Release Msg limiter correctly if pipeline.Start fails (GH #348)

### Misc

* config/tls: Fix tls_client parsing (Thanks @AluisioASG!)

### Documentation

* docs: Replace foxcpp.dev/maddy with maddy.email
* docs: Remove reference to local_modifiers from multiple-domains.md
* docs: fixed small error (Thanks @0xflotus!)
* Fix a typo in maddy-smtp man doc (Thanks @Defman21!)

### Docker

This release changes how Docker image is built reverting default directories
behavior to what 0.4.2 did but implemented in a different way so we have
the best of both worlds.

### build.sh

* Allow setting build tags
* Do not try to install man pages if they were not built

### 3rd party libraries

* go-smtp now uses Postfix success responses
* go-msgauth/dkim now supports both RSA public key formats
  (see https://github.com/emersion/go-msgauth/issues/43)

<hr /><em>This discussion was created from the release <a href='https://github.com/foxcpp/maddy/releases/tag/v0.4.4'>maddy 0.4.4</a>.</em>

**Comment by z3ugma** [comment]
@foxcpp nice, thanks for the release. 

I notice the binaries are labeled `0.4.3` while the release is called `0.4.4`, is this just a misnaming?

Oh also, for the install, are we to use `zstd` to unarchive the `.tar.zst` file, and then move the whole `maddy-0.4.3+cd1d527-x86_64-linux-musl` into `/etc/`?

**Reply by foxcpp** [reply]
Oh snap, must be misnaming. I will take a look.

**Reply by foxcpp** [reply]
I'm really sorry. I actually accidentally uploaded 0.4.3 tarballs I had lying around :(
Fixed it now.
